requirements.txt

```diff
@@ -6,3 +6,4 @@ numpy==1.19.2
 pep8-naming==0.3.3
 pep8==1.7.1
 zipp==0.5.0
+regex==2020.10.28
```

Comment:
Any reason to choose `regexp` over built-in `re`? =)

Reply:
Done.